modules/storages/app/common/storages/peripherals/nextcloud_connection_validator.rb

```diff
@@ -59,6 +60,17 @@ def query
                      .call(storage: @storage, auth_strategy:)
       end
 
+      def files
```

Comment:
🟡 minor, but if we have now a meaningful name to distinguish outbound requests, can we rename the other query to `capabilities`?